services/shopping_list.js

```diff
@@ -0,0 +1,98 @@
+const axios = require('axios');
+/*
+    TODO: 
+    INGREDIENT NAME SHOULD MATCH PRODUCT NAME
+    IN THE EVENT THE USER PUTS RICE AS THE INGREDIENT
+    BUT IT'S EXPECTING A DIFF RICE AND HAS TWO DIFF PRODUCTS
+    THE FUNCTION MIGHT PICK UP AN EXISTING RICE ON THE CURR PANTRY
+    THAT MIGHT NOT BE NECESSARILY THE TYPE OF RICE THE USER NEEDS 
+    AND NOT TELL THE USER THEY NEED THAT SPECIFIC PRODUCT, WHEN THEY ACTUALLY DO
+
+    TO AVOID, ADV USER'S TO EITHER NAME THE INGREDIENTS THE SAME THEIR PRODS ARE NAMED
+    OR IF USING A CONSISTENT PROD FOR, LET'S SAY 'RICE', SPECIFY WHEN THEY ACTUALLY WANT 
+    A DIFF ONE ON THE RECIPE'S ING
+
+    GIVEN THE OBS UP TOP, CONSIDER MAKING THE INGREDIENTS NAME UNIQUE
+    THAT WAY 2 'RICE' ROWS WON'T BE MADE
+    USERS WILL HAVE TO NAME THEM DIFF SINCE THEY'LL REPRESENT DIFF PRODUCTS
+*/
+
+
+const getAllWeeklyMeals = async _ => {
+    const weekMealsCall = await axios.get('http://localhost:11235/mealSchedule/user/1');
```

Comment:
as we discussed try using `MealScheduleServices.getScheduledMeals(1)` directly instead of making another web request

Reply:
You may need to figure out how to import those services into your file.